modules/wizard/wizard.php

```diff
@@ -183,6 +183,13 @@ public function setup_wizard_page() {
 			exit;
 		}
 
+		$home_page_id = get_option( 'page_on_front' );
+		$home_page_language = $this->model->post->get_language( $home_page_id );
+
+		if ( ! $home_page_language && ! in_array( $this->step, array( 'licenses', 'languages', 'media', 'untranslated-contents' ) ) ) {
```

Comment:
I propose to check `$this->model->get_objects_with_no_lang( 1 )` rather than only the language of the home page. Because the issue is the home page language only for the homepage step, but for other steps, the issue is fr other pages (WC pages).

Reply:
I did like that at the beginning before pushing the first time 😁